python/dgl/nn/pytorch/conv/gcn2conv.py

```diff
@@ -0,0 +1,272 @@
+    It is defined by :math:`\beta_l = \log(\frac{\lambda}{l}+1)\approx\frac{\lambda}{l}`,
+    where :math:`\lambda` is a hyperparameter. :math: `\beta` ensures that the decay of
+    the weight matrix adaptively increases as we stack more layers.
+
+    Parameters
+    ----------
+    in_feats : int
+        Input feature size; i.e, the number of dimensions of :math:`h_j^{(l)}`.
+    layer : int
+        the index of current layer.
+    alpha : float
+        The fraction of the initial input features. Default: ``0.1``
+    lambda_ : float
+        The hyperparameter to ensure the decay of the weight matrix
+        adaptively increases. Default: ``1``
+    project_initial_features : bool
+        Whether to share a weight matrix between initial features and
+        smoothed features. Default: ``True``
+    bias : bool, optional
+        If True, adds a learnable bias to the output. Default: ``True``.
+    activation : callable activation function/layer or None, optional
+        If not None, applies an activation function to the updated node features.
+        Default: ``None``.
+    allow_zero_in_degree : bool, optional
+        If there are 0-in-degree nodes in the graph, output for those nodes will be invalid
+        since no message will be passed to those nodes. This is harmful for some applications
+        causing silent performance regression. This module will raise a DGLError if it detects
+        0-in-degree nodes in input graph. By setting ``True``, it will suppress the check
+        and let the users handle it by themselves. Default: ``False``.
+
+    Note
+    ----
+    Zero in-degree nodes will lead to invalid output value. This is because no message
+    will be passed to those nodes, the aggregation function will be appied on empty input.
+    A common practice to avoid this is to add a self-loop for each node in the graph if
+    it is homogeneous, which can be achieved by:
+
+    >>> g = ... # a DGLGraph
+    >>> g = dgl.add_self_loop(g)
+
+    Calling ``add_self_loop`` will not work for some graphs, for example, heterogeneous graph
+    since the edge type can not be decided for self_loop edges. Set ``allow_zero_in_degree``
+    to ``True`` for those cases to unblock the code and handle zero-in-degree nodes manually.
+    A common practise to handle this is to filter out the nodes with zero-in-degree when use
+    after conv.
+
+    Examples
+    --------
+    >>> import dgl
+    >>> import numpy as np
+    >>> import torch as th
+    >>> from dgl.nn import GCN2Conv
+
+    >>> # Homogeneous graph
+    >>> g = dgl.graph(([0,1,2,3,2,5], [1,2,3,4,0,3]))
+    >>> feat = th.ones(6, 3)
+    >>> g = dgl.add_self_loop(g)
+    >>> conv1 = GCN2Conv(3, layer=1, alpha=0.5, \
+                project_initial_features=True, allow_zero_in_degree=True)
+    >>> conv2 = GCN2Conv(3, layer=2, alpha=0.5, \
+                project_initial_features=True, allow_zero_in_degree=True)
```

Comment:
```suggestion
    >>> conv1 = GCN2Conv(3, layer=1, alpha=0.5, \
    ...         project_initial_features=True, allow_zero_in_degree=True)
    >>> conv2 = GCN2Conv(3, layer=2, alpha=0.5, \
    ...         project_initial_features=True, allow_zero_in_degree=True)
```

The line continuations should start with `...` like what you see in Python console.